doc/releases.rst

```diff
@@ -5,173 +5,173 @@ Ceph Releases
-+----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
-| September 2013 |`0.69`_    |           |           |           |           |           |              |           |           |
-|                +-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
-|                |`0.68`_    |`0.67.3`_  |           |           |           |           |              |           |           |
-+----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
-| August    2013 |           |`0.67.2`_  |           |           |           |           |              |           |           |
-|                +-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
-|                |           |`0.67.1`_  |           |           |           |           |              |           |           |
-|                +-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
-|                |           |`0.67`_    |           |           |           |           |              |           |           |
-+----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
-
-
-.. _12.1.2: ../release-notes#v12-1-2-luminous-rc
-.. _12.1.1: ../release-notes#v12-1-1-luminous-rc
-.. _12.1.0: ../release-notes#v12-1-0-luminous-rc
-.. _12.0.3: ../release-notes#v12-0-3-luminous-dev
-.. _12.0.2: ../release-notes#v12-0-2-luminous-dev
-.. _12.0.1: ../release-notes#v12-0-1-luminous-dev
-.. _12.0.0: ../release-notes#v12-0-0-luminous-dev
++----------------------+-------------+-----------+----------+---------------+-----------+----------+------------+------------+-------------+
+|                      | `Luminous`_ | `Kraken`_ | `Jewel`_ | `Infernalis`_ | `Hammer`_ | `Giant`_ | `Firefly`_ | `Emperor`_ | `Dumpling`_ |
+|                      |  LTS        | Stable    | LTS      | Stable        | LTS       | Stable   | LTS        | Stable     | LTS         |
++----------------------+-------------+-----------+----------+---------------+-----------+----------+------------+------------+-------------+
+| First release        | August      | January   | April    | November      | April     | October  | May        | November   | August      |
+|                      | 2017        | 2017      | 2016     | 2015          | 2015      | 2014     | 2014       | 2013       | 2013        |
++----------------------+-------------+-----------+----------+---------------+-----------+----------+------------+------------+-------------+
+| Estimated retirement |             | August    | November |               | May       |          | January    |            | March       |
```

Comment:
Move Infernalis' "August 2017" down to the next row for "Actual Retirement" -- also add "August 2017" for Hammer under the "Actual Retirement" row

Reply:
ack, does this actually warrant a ceph-users/devel announcement?

Reply:
Probably would be nice even though this page states that only the most recent two LTSs will be maintained -- @liewegas?

Reply:
@dillaman You meant Kraken, not Infernalis.

Reply:
I don't remember us ever announcing when a release went EOL, but that doesn't mean we didn't :-/

Reply:
@smithfarm Indeed -- thanks for keeping me honest. :-p

Reply:
let us start now then :) we announced that hammer would be EOL when L is released, we didn't explicitly announce retirals so far (Back when backports started Loic did one for Dumpling iirc)

Reply:
The best place for the hammer, kraken EOL announcement is/would have been in the Luminous release announcement . . .

Reply:
Yeah I don't think we have been sending EOL announcements, but we should probably start :).

A separate email seems better to me (with EOL in the subject) to catch people's attention. (Next time we can *also* mention it in the LTS release too, but whatever.)